### PR TITLE
Add rastervision-core

### PR DESCRIPTION
**Pull request by psortos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/rastervision-core/0001-fix-imports.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+index 6bcd880..3b8dd4b 100644
+--- a/setup.py
++++ b/setup.py
+@@ -31,4 +31,10 @@ setup(
+     'raster deep-learning ml computer-vision earth-observation geospatial geospatial-processing',
+     packages=find_namespace_packages(exclude=['integration_tests*', 'tests*']),
+     install_requires=install_requires,
+-    zip_safe=False)
++    zip_safe=False,
++    entry_points = {
```
Comment (psortos) [inline recipes/rastervision-core/0001-fix-imports.patch:28]: I added an entry point in this patch due to an issue with an executable file which was causing the build to fail

File: recipes/rastervision-core/0001-fix-imports.patch [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
++shapely==1.6.*
++pillow==8.3.*
++pyproj==2.6.*
++rasterio==1.2.6
++scikit-learn==0.24.*
++imageio==2.3.*
++pystac==0.5.6
++mask-to-polygons==0.0.2
++opencv-python
+\ No newline at end of file
```
Comment (psortos) [inline recipes/rastervision-core/0001-fix-imports.patch:17]: I found two different requirements folders for this repo
- one in the main rastervision repo https://github.com/psortos/staged-recipes/tree/add-rastervision-core/recipes/rastervision-core 
- another in the pypi pacakge for rastervision-core https://files.pythonhosted.org/packages/40/d0/4ff1c4c5b31b443deeffff1ce48f5bf03f979575e4b8726b121062180d8a/rastervision_core-0.13.1.tar.gz 
This requirements removes some of the version requirements which were creating conflicts

**Comment by psortos** [comment]
@conda-forge/staged-recipes ready for review

**Comment by ocefpaf** [comment]
I noticed that you are removing recipes in your PR. That is handled automatically and manual removals can be troublesome. Can you revert those commits? If that is to difficult maybe you can re-submit this PR from a fresh branch?

**Pull request merged**